### PR TITLE
Command to remove orphaned webauthn credentials

### PR DESCRIPTION
**Pull request by kesselb**

### CODE REVIEW

File: lib/Command/CleanUp.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		try {
+			$result = $qb->executeQuery();
+
+			while ($row = $result->fetch()) {
+				$userIds[] = $row['user_handle'];
+			}
+
+			$result->closeCursor();
+		} catch (\Exception $exception) {
+			// what could go wrong ;)
```
Comment (ChristophWurst) [inline lib/Command/CleanUp.php:110]: I'd just fail hard. There is no point in continuing

File: lib/Command/CleanUp.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$io = new SymfonyStyle($input, $output);
+		$io->title('Find orphaned webauthn credentials');
+		$io->text([
+			'This command search webauthn credentials for deleted users.',
+			'Found credentials for deleted users will be removed.'
+		]);
+
+		$confirm = $io->confirm('Start process?', true);
```
Comment (ChristophWurst) [inline lib/Command/CleanUp.php:71]: do we really need this? :see_no_evil: 

I'd say if the admin calls this command they should know about the consequences

Comment (kesselb) [reply lib/Command/CleanUp.php:71]: > should know about the consequences

Good point indeed. I forgot to add a description for the command :see_no_evil: 

> do we really need this? see_no_evil

I don't have a opinion :man_shrugging: Do you want me to remove it?

File: lib/Command/CleanUp.php (hunk 1 above)
Comment (kesselb) [reply lib/Command/CleanUp.php]: OK

File: lib/Command/CleanUp.php (hunk 2 above)
Comment (kesselb) [reply lib/Command/CleanUp.php]: Confirm is gone.

File: lib/Command/CleanUp.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+					$this->webauthnMapper->deletePublicKeyCredentialsByUserId($userId);
+				} catch (Exception $e) {
+					$io->caution('Error deleting credentials: ' . $e->getMessage());
+				}
+			}
+		}
+
+		$io->success('Orphaned webauthn credentials removed.');
+
+		$io->text('Thank you for using Two-Factor WebAuth!');
```
Comment (st3iny) [inline lib/Command/CleanUp.php:82]:
```suggestion
		$io->text('Thank you for using Two-Factor WebAuthn!');
```

**Review by st3iny — APPROVED** [review]
Tested and works. The styling looks fancy.

**Pull request merged**